build.sc

```diff
@@ -61,7 +62,7 @@ trait AmmInternalModule extends mill.scalalib.CrossSbtModule{
         Nil
 
     val extraDir2 = PathRef(
-      if (sv == "2.12.10" || sv == "2.13.1") millSourcePath / "src" / "main" / "scala-2.12.10-2.13.1+"
+      if (isScala2_12_10OrLater(sv)) millSourcePath / "src" / "main" / "scala-2.12.10-2.13.1+"
```

Comment:
I think `2.13.0` has to be explicitly excluded here (the extra dir is for `2.12.{10, 11, …}`, and `2.13.{1, …}` excluding `2.13.0`).

Reply:
The variable name may be confusing, the set does not contain 2.13.0

Reply:
Oh my bad 😬